docs/ibc/light-clients/client-state.md

```diff
@@ -1,3 +1,65 @@
 <!--
 order: 2
--->
\ No newline at end of file
+-->
+
+# Implementing the ClientState interface
+
+Learn how to implement the [Client State](https://github.com/cosmos/ibc-go/blob/main/modules/core/exported/client.go#L36) interface.
+
+### ClientType
+
+`ClientType` should return a unique string identifier of the light client.
```

Comment:
```suggestion
`ClientType` should return a unique string identifier of the light client. This will be used when generating a client identifier. The format is created as follows: `ClientTypes-{N}` where `{N}` is the unique global nonce associated with a specific client. 
```

Or something like that